model/storage_key_encryption_key.rb

```diff
@@ -0,0 +1,21 @@
+# frozen_string_literal: true
+
+require_relative "../model"
+
+class StorageKeyEncryptionKey < Sequel::Model
+  plugin :column_encryption do |enc|
+    enc.column :key
+    enc.column :init_vector
+  end
+
+  def to_hash
```

Comment:
I think to_hash is a bit innocuous and possibly called by programs as a bit of an implied protocol, how about something like "key_material_hash"? If used rarely, you can even belabor the point: `secret_key_material_hash`. Brevity is more at issue for something done in a lot of sites.